WebRoot/js/zimbraMail/calendar/view/ZmCalendarPrefsPage.js

```diff
@@ -791,6 +791,11 @@ function(templateId) {
     radioNormal = new DwtRadioButton({parent:this, name:radioName, parentElement:(this._htmlElId + "_CAL_WORKING_HOURS_NORMAL")});
     radioNormal.setSelected(!isCustom);
     var radioNormalId = radioNormal.getInputElement().id;
+
+    // Accessibility
+    radioNormal.getInputElement().setAttribute('aria-label', ZmMsg.normal);
+    radioNormal.getInputElement().removeAttribute('aria-labelledby');
```

Comment:
Instead of invoking getInputElement multiple times better store the return value in a variable?
Also is there a possibility of the value being undefined?

Reply:
Done. The radio component must have input element so it will never be undefined.